source/manuals/programming-languages/html.html.md.erb

```diff
@@ -1,85 +1,162 @@
 
-[Demo of how landmark roles help screen reader users](http://tink.co.uk/2011/07/how-do-aria-landmark-roles-help-screen-reader-users/)
+A "Skip to main content" link should be added to help screen reader and keyboard
+users skip past your general site navigation and get straight to the content.
+You can use the [GOV.UK skip link component from the design system](https://design-system.service.gov.uk/components/skip-link/)
+for this.
 
-## Main content
+### Sectioned content
 
-The `<main>` tag should be used to identify the main content of your page. Support for it is still marginal so ensure to add the `role="main"` attribute.
+Use the `<section>` tag to represent standalone, thematic groupings of content
+where a more specific semantic element would not be appropriate.
 
-## Sectioned content
+Examples of sections would be chapters, or various tabbed pages in a tabbed dialog box,
+or a home page split into sections such as introduction, news items and contact information.
 
-When using the `<section>` tag, add an `aria-labelledby` attribute linking to the id of the heading that identifies that section.
+You should add an `aria-labelledby` attribute linking to the id of the heading
+that identifies that section.
 
 Example:
 
-    <section aria-labelledby="advice">
-      <h1 id="advice">Help and advice</h1>
+```html
+<section aria-labelledby="introduction">
+    <h1 id="introduction">Introduction</h1>
+    <p>People have been catching fish for food since before recorded history...</p>
+</section>
 
-## Navigational elements
+<section aria-labelledby="equipment">
+    <h1 id="equipment">Equipment</h1>
+    <p>The first thing you'll need is a fishing rod or pole that you find comfortable
+    and is strong enough for the kind of fish you're expecting to try to land...</p>
+</section>
+```
 
-When using the `<nav>` tag, add the ARIA landmark `role="navigation"`.
+### Navigational elements
 
-The detail of using the `<nav>` tag is covered in this [blog post](https://insidegovuk.blog.gov.uk/2013/07/03/rethinking-navigation/).
+Use `<nav role="navigation">` to wrap groups of links that aren't already in
+another context (e.g. `<footer>`). Common examples of navigation sections are
+menus, tables of contents, and indexes.
 
-### Emphasis
+As with `<section>`, it is advisable to use `aria-labelledby` to label your `<nav>`.
+This helps to distinguish different `<nav>` blocks in the page.
 
-If you want to give the current location emphasis within a navigational list, use a `<strong>` tag.
+The GOV.UK blog has [more information regarding the use of the `<nav>` tag](https://insidegovuk.blog.gov.uk/2013/07/03/rethinking-navigation/).
 
-Example:
+### Aside
+
+`<aside role="complementary">` should be used for content related to the primary content
+of the webpage, but which does not constitute the primary content of the page.
+
+Examples include author information, related links and related content.
```

Comment:
The examples should be as clear as possible, especially as when to use `aside` is confusing to a lot of people. The most typical example is a sidebar. Something like "author information" could lead to people using it wrongly.
(Not a blocker.)

Reply:
`aside` _is_ confusing. It can be used for sidebars, but should only be used if the content of the sidebar is "tangentially related to the content of the parenting sectioning content". So if it's just a sidebar of adverts or newsletter signup forms - as sidebars often are - then `aside` would probably not be the most appropriate choice.

Think this might be worth following up in a separate issue. 👍